plugins/mediatek-scaler/mediatek-scaler.quirk

```diff
@@ -0,0 +1,4 @@
+[PCI\VID_1028&PID_0C88]
+Flags = probe-vcp
+[PCI\VID_1028&PID_0C8A]
```

Comment:
Probing C8 should be safe for everything I'd expect. It's the stuff _after that_ which is risky.

Although an unlikely case - this will still potentially break if someone connects another monitor with Mediatek soc into the supported AIO I'd think.

Reply:
Yeah, testing the controller type is still required. Introduced `fu_mediatek_scaler_device_verify_controller_type` see if it helps.